### ISSUE TITLE
EEGLAB version 2024.0 - missing folder plugins

### ISSUE THREAD

**Issue by Filipe007** [eeglabdevelopers/eeglab#749]
### Description

Dear all,

Can you please update EEGLAB version 2024.0 as the plugins folder is missing

<br>
#### Steps to Reproduce

1. On matlab online download new version EEGLAB version 2024.0
2. If you try to download plugins it does not work
3. Issue related to missing folder

<br>
#### Expected behavior:
The EEGLAB GUI should be able to download plugins

<br>
#### Actual behavior:
Not able to download

<br>
#### How to fix it:
1) Go to folder /MATLAB Drive/FileExchange/eeglab-2024.0
2) Go create subfolder plugins

<br>
#### Versions

|   |   |
|----| ---- | 
| OS version     | [MacOS 13.6.3 ] |
| Matlab version | [23.2.0.2521687 (R2023b) Update 7] |
| EEGLAB version | [eeglab-2024.0] |

**Comment by arnodelorme** [comment]
Yes, this is an issue with the Mathworks solution. We will look into it. It is better to download EEGLAB on this page https://sccn.ucsd.edu/eeglab/download.php
